entity-service/src/main/kotlin/com/egm/stellio/entity/authorization/Neo4jAuthorizationRepository.kt

```diff
@@ -0,0 +1,65 @@
+package com.egm.stellio.entity.authorization
+
+import com.egm.stellio.entity.model.Property
+import com.egm.stellio.entity.model.Relationship
+import org.neo4j.ogm.session.Session
+import org.springframework.stereotype.Component
+
+@Component
+class Neo4jAuthorizationRepository(
+    private val session: Session
+) {
+
+    data class AvailableRightsForEntity(
+        val targetEntityId: String,
+        val right: Relationship? = null,
+        val grpRight: Relationship? = null
```

Comment:
Can't they be merged? Because, at the end, what we want to know is the list of rights an user has on an entity, whether it is directly or via a group membership (and the type is just what we want?).

Reply:
Yes indeed they can be merged into a list.

Reply:
And indeed we need only the type for the moment.